gradle.properties

```diff
@@ -2,14 +2,15 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Mod Properties
-mod_version=rv9
+mod_version=1.10+1.17-fabric
```

Comment:
I would like to stick with my versioning scheme. I understand it's not standard, I know semantic versioning, I use it with basically all of my other projects. I'm just stubborn :P

This project is kind of old so I didn't follow it back then when I wrote it. And it would slightly irk me to see the version scheme change in the list of versions on CF. I know, not great reasons :) But I would prefer it that way.

Also, I don't like having the MC version in the mod version, just feel it's too busy. That's why I had the MC version in the `archiveAppendix` in the `build.gradle` (also because I couldn't figure out a good way to use variables to sub in the MC version, I hardcoded it)

Reply:
Starting the version number with 'rv' causes a warning to be produced in the game log (from the fabric loader I think). If there were no runtime warnings I would not have changed it (since this mod has no API to speak of). 

```
[main/WARN]: Mod `bowinfinityfix` (rv9) does not respect SemVer - comparison support is limited.
```

Up to you to decide if silencing that warning is enough to change your mind

Reply:
Yeah, I've seen that line. One line in a log file that most users will not read isn't enough to sway me :)

From what I can tell, this deals with like update checking or ensuring client<->server are network compatible, or something like that? This mod almost always only gets one version per major MC version, so that doesn't matter to me.